ironfish-rust-nodejs/index.d.ts

```diff
@@ -320,6 +320,7 @@ export namespace multisig {
     incomingViewKey: string
     outgoingViewKey: string
     proofAuthorizingKey: string
+    ledger: boolean
```

Comment:
This is an unused change and will be deleted the next time this file is re-generated. This is auto-generated by NAPI-RS so you cannot modify it manually.